splinterd/src/config/builder.rs

```diff
@@ -23,15 +23,23 @@ pub trait PartialConfigBuilder {
     fn build(self) -> Result<PartialConfig, ConfigError>;
 }
 
-fn get_file_path(cert_dir: &str, file: &str) -> String {
+fn get_tls_file_path(config_dir: &str, cert_dir: &str, file: &str) -> String {
     if file.starts_with("./") || file.starts_with("../") {
```

Comment:
Should this check for startswith('/') too? I think yes, in that if the user specified the full path "/tmp/foobar" then we don't want to prefix it with anything.

Reply:
That is addressed in the second commit